sycl/test-e2e/KernelCompiler/sycl_time_trace.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 
```

Comment:
```suggestion
// REQUIRES: (opencl || level_zero)
```

This test also had this requires before the RTC patch, which is why it failed in the CI